analytics/dev/django.Dockerfile

```diff
@@ -16,5 +16,13 @@ ENV PYTHONUNBUFFERED 1
 COPY ./setup.py /opt/django-project/setup.py
 RUN pip install --editable /opt/django-project[dev]
 
+
+# Install spack
+RUN git clone -c feature.manyFiles=true https://github.com/spack/spack.git /opt/spack
```

Comment:
Just wanted to point out, the version of `spack` used in the job processor will be whatever the status of `develop` is at build time of the docker image. Not sure if that matters or not, but figured it was worth mentioning.